calico_kubernetes/tests/policy_test.py

```diff
@@ -135,18 +135,15 @@ def test_parse_icmp_rule(self):
         # Assert on the result
         assert_equal(parsed['action'], "allow")
         assert_equal(parsed['protocol'], "icmp")
-        assert_equal(parsed['icmp_type'], 8)
+        assert_equal(parsed['icmp_type'], '8')
         assert_equal(parsed['src_net'], IPNetwork("1.2.0.0/16"))
 
     @parameterized.expand([
         ("allow icmp type code 4"),
-        ("allow icmp type ten code 4"),
```

Comment:
calico_kubernetes uses a specific version of libcalico, not master, therefore, the libcalico validation changes are not reflected in these tests. Should we remove the version locking of libcalico or should we remove these test cases?